### DISCUSSION TITLE
Contributor badges and names

### DISCUSSION THREAD

**Discussion by Warrobot10** [monkeytypegame/monkeytype#4544] [Ideas] [answered]
I saw that auto change logs were recently implemented, and thought that a workflow could be added that runs every week or something, to add people who have contributed to monkeytype to get the Contributor badge and username in the about page. Thanks!

**Comment by Miodec** [comment, marked as answer]
Ive got a script that updates the contributors list that i run from time to time

**Reply by Warrobot10** [reply]
Oh, okay then. Then perhaps that could be used in conjunction with the new change logs every month or so. Thanks!
